demo/src/app/components/+pagination/demos/page-changed-event/page-changed-event.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-pagination-page-changed-event',
+  templateUrl: './page-changed-event.html'
+})
+export class DemoPaginationPageChangedComponent {
+  currentPage = 4;
+
+  pageChanged(event: any): void {
+    console.log('Page changed to: ' + event.page);
```

Comment:
get rid of `console.log` and show event data in `<pre>` or similar (like you do in `numPages` demo )